scripts/ccf/az-cleanroom-aci/up.sh

```diff
@@ -43,6 +43,8 @@ az-cleanroom-aci-up() {
     sudo cp $KMS_SERVICE_CERT_PATH /usr/local/share/ca-certificates/kms_ca.crt
```

Comment:
cool. So we no longer have to pass service_cert in curl

Reply:
You raise a good point, I had a think about it and made a separate PR since it's unrelated to this change.
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/264